<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventory</class>
 <widget class="QWidget" name="Inventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="numGrass">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numWood">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numStone">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numLeaf">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numDirt">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numSand">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>INVENTORY</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonGrass">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>GRASS</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonWood">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>WOOD</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonStone">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>STONE</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonLeaf">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>LEAF</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDirt">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>DIRT</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QRadioButton" name="radioButtonSnow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>SNOW</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="numSnow">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QRadioButton" name="radioButtonSand">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>SAND</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QRadioButton" name="radioButtonMushStem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>MUSHROOM
STEM</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="numMushroomStem">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="numCactus">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonCactus">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>CACTUS</string>
   </property>
  </widget>
  <widget class="QFrame" name="cactusFrame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="numIce">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonIce">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>ICE</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSandstone">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>SANDSTONE</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_10">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>111</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QLCDNumber" name="numSandstone">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="radioButtonMushHat">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="text">
    <string>MUSHROOM
HAT</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_11">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>131</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QLCDNumber" name="numMushroomHat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
  </widget>
  <zorder>frame_8</zorder>
  <zorder>frame_11</zorder>
  <zorder>frame_10</zorder>
  <zorder>radioButtonSandstone</zorder>
  <zorder>frame_9</zorder>
  <zorder>frame_6</zorder>
  <zorder>cactusFrame</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame_4</zorder>
  <zorder>frame_3</zorder>
  <zorder>frame_7</zorder>
  <zorder>frame_5</zorder>
  <zorder>frame</zorder>
  <zorder>numGrass</zorder>
  <zorder>numWood</zorder>
  <zorder>numStone</zorder>
  <zorder>numLeaf</zorder>
  <zorder>numDirt</zorder>
  <zorder>numSand</zorder>
  <zorder>label</zorder>
  <zorder>radioButtonGrass</zorder>
  <zorder>radioButtonWood</zorder>
  <zorder>radioButtonStone</zorder>
  <zorder>radioButtonLeaf</zorder>
  <zorder>radioButtonDirt</zorder>
  <zorder>numCactus</zorder>
  <zorder>radioButtonCactus</zorder>
  <zorder>numIce</zorder>
  <zorder>radioButtonIce</zorder>
  <zorder>radioButtonMushHat</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
